src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -573,6 +580,7 @@ public async Task<ActionResult<ApiResourceAllocationRequest>> ApproveProjectAllo
                 {
                     if (result.OrgPositionId.HasValue)
                         or.OrgChartPositionWriteAccess(result.Project.OrgProjectId, result.OrgPositionId.Value);
+                    or.BeRequestCreator(requestId);
```

Comment:
Hmm, it kind of make sense, but it kind of doesnt.. I think we might have to offload this to the logic? 
Just seem like we need to take into account what type the workflow is.

Ex. now i can do the "propose" from owner side by hitting this endpoint, and if i created the request as a task owner, i can also propose.. I think these approve steps must be delt with separately.